packages/apps-config/src/links/statescan.ts

```diff
@@ -18,7 +18,6 @@ export const Statescan: ExternalDef = {
     'Kusama People': 'people-kusama',
     'Laos Network': 'laos',
     Litentry: 'litentry',
```

Comment:
Should we remove this litentry?

Reply:
Update in #11295.

Reply:
Thanks @wliyongfeng 

I think this PR can be merged now, along with #11295 🙌